source/fundamentals/encrypt-fields.txt

```diff
@@ -0,0 +1,70 @@
+==============
+Encrypt Fields
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can encrypt fields in a document using a set of features called
+**in-use encryption**.
+
+In-use encryption enables your client applications to encrypt data
+*before* sending it to MongoDB, and to query documents with encrypted fields.
+
+Because the driver encrypts the data before sending it to MongoDB, only
+your configured client applications can decrypt the data. Only applications
+using the driver with access to your encryption keys can access the decrypted,
+plaintext data. Should you have unauthorized access to your database, an
+attacker could only see the encrypted, ciphertext data.
+
+In-use encryption can help prevent exposure of the following sensitive types of data:
+
+- Credit card numbers
+- Addresses
+- Health information
+- Financial information
+- Any other sensitive or personally identifiable information (PII)
+
+MongoDB offers the following ways to encrypt fields:
+
+{+qe+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+{+qe+} is the next-generation in-use encryption feature,
+introduced in MongoDB 6.0 and available as a public preview. {+qe+}
+supports searching encrypted fields for equality and encrypts each value
+uniquely.
+
+The MongoDB manual contains detailed information on the following {+qe+} topics:
+
+- Begin a working {+qe+} example with the :v6.0:`queryable-encryption quick start </core/queryable-encryption/quick-start>`.
```

Comment:
Issue: I think this is a case in which the title of the target content should be used since it doesn't necessarily describe the content. According to the style guide:
"If existing text can’t sufficiently describe the target content, create a cross-reference sentence for the link. For the link text, use the title of or a heading in the target content, if possible."
See https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text

Applies to all instances